packages/components/tests/integration/components/hds/alert/index-test.js

```diff
@@ -18,6 +19,12 @@ module('Integration | Component | hds/alert/index', function (hooks) {
   test('it should render the component with a CSS class that matches the component name', async function (assert) {
     await render(hbs`<Hds::Alert @type="inline" id="test-alert" />`);
     assert.dom('#test-alert').hasClass('hds-alert');
+
+    if (shouldForceAudit()) {
+      // `&enableA11yAudit` set in the URL
+      await a11yAudit();
+    }
+    assert.ok(true, 'no a11y errors found in default component invocation');
```

Comment:
It seems like this should be inside the conditional otherwise no tests for a11y were actually run?

Reply:
a11y errors are thrown within `a11yAudit` so I guess the logic here is: if no errors were thrown we can log a success message; I _think_ an `assert` inside a conditional statement wouldn't be valid anyway